llvm/test/CodeGen/AMDGPU/local-accessor-to-shared-memory-triple.ll

```diff
@@ -0,0 +1,25 @@
+; This test checks that the Local Accessor to Shared Memory pass runs with the
+; `amdgcn-amd-amdhsa` triple, but not with `amdgcn-amd-amdpas`.
```

Comment:
```suggestion
; `amdgcn-amd-amdhsa` triple, but not with `amdgcn-amd-amdpal`.
```

I also suggest to remove `llvm/test/CodeGen/AMDGPU/local-accessor-to-shared-memory-invalid-triple.ll`, because it is included into this test

Reply:
Agreed, removing now.